scripts/eval/run_and_process_tortoize.py

```
@@ -0,0 +1,149 @@
+from loguru import logger
+from pandas import DataFrame
+
+from sampleworks.eval.eval_dataclasses import Trial
+from sampleworks.eval.grid_search_eval_utils import parse_eval_args, setup_evaluation_parameters
+
+
+# TODO make more general: https://github.com/diff-use/sampleworks/issues/93
+def main(args) -> None:
+    # check that phenix is installed and available, bail early if not.
+    try:
+        subprocess.call("tortoize", stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)